portals/publisher/src/main/webapp/source/src/app/components/GlobalPolicies/Listing/Listing.tsx

```diff
@@ -0,0 +1,1115 @@
+    const theme : any = useTheme();
+    const { globalPolicyAddIcon } = theme.custom.landingPage.icons;
+    const intl = useIntl();
+
+    /**
+     * Empty array to store the policies
+     * @param {Policy[]} inputPolicies - Policies.
+     * @returns {Policy[]} - Initial object array to hold selected gateways from the dropdown.
+     */
+    const getInitialSelectedGateways = (inputPolicies: Policy[]): SelectedGateway[] => {
+        const selectedGatewaysList: SelectedGateway[] = [];
+    
+        inputPolicies.forEach((policy) => {
+            const { id } = policy;
+            const selectedGatewayValue: SelectedGateway = {
+                id,
+                gatewayLabels: [],
+            };
+            selectedGatewaysList.push(selectedGatewayValue);
+        });
+
+        return selectedGatewaysList;
+    };
+
+    /**
+     * After deploying or undeploying, we need to clean the selected gateways list.
+     * @param {string} policyId - Policy ID.
+     * @returns {SelectedGateway[]} - Deployed policies' selected gateways should be empty. 
+     */
+    const cleanSeletectedGateways = (policyId: string): SelectedGateway[] => {
+        const updatedSelectedGateways: SelectedGateway[] = selectedGateways.map((data) => {
+            if (data.id === policyId) {
+                return {
+                    ...data,
+                    gatewayLabels: [],
+                };
+            }
+            return data;
+        });
+        
+        return updatedSelectedGateways;
+    };
+
+    /**
+     * Fetch the data from the backend to the compoenent.
+     */
+    const fetchGlobalPolicies = () => {
+        setLoading(true);
+        // Due to a bug in current backend, we pass 0, 10 as a workaround for now
+        const promisedPolicies = API.getAllGatewayPolicies();
+        promisedPolicies
+            .then((response: any) => {
+                setPolicies(response.body.list);
+                setSelectedGateways(getInitialSelectedGateways(response.body.list));
+                setLoading(false);
+            })
+            .catch((/* error */) => {
+                // console.error(error);
+                APIMAlert.error(intl.formatMessage({
+                    id: 'Fetching.Policies.Error',
+                    defaultMessage: 'Error while fetching policies',
+                }));
+                setnotFound(true);
+                setLoading(false);
+            })
+    };
+
+    /**
+     * Fetch environements publisher settings from the backend to the compoenent.
+     * This data has been used to get the full gateway environment list.
+     */
+    const fetchSettings = () => {
+        setLoading(true);
+        const promisedSettings = API.getSettings();
+        promisedSettings
+            .then((response: any) => {
+                setEnvironments(response.environment);
+            })
+            .catch((/* error */) => {
+                // console.error(error);
```

Comment:
Let's remove the commented out code

Reply:
Addressed the required changes by [fddf57f](https://github.com/wso2/apim-apps/pull/515/commits/fddf57fa7308a53094c808e9bf2d2b40ed71b644) and [0ad490c](https://github.com/wso2/apim-apps/pull/515/commits/0ad490c497b08f999babbe521feaf83d029a6698)